backend-rust/src/indexer.rs

```diff
@@ -2022,17 +2097,33 @@ impl PreparedRejectModuleTransaction {
     }
 }
 
+/// The token name is generated by using a `version byte 2` and concatenating it
+/// with the leb128 byte encoding of the contract index and the leb128 byte
+/// encoding of the contract subindex concatenated with the token id in bytes.
+/// Finally the whole byte array is base 58 check encoded.
+/// https://proposals.concordium.software/CIS/cis-2.html#token-address
+/// The token address is a unique identifier used to distinguish tokens across
+/// all smart contracts.
+pub fn get_token_address(
+    contract_address: ContractAddress,
+    token_id: TokenId,
+) -> Result<String, anyhow::Error> {
+    Ok(format!("{}", TokenAddress::new(contract_address, token_id)))
+}
```

Comment:
Not sure this function is justified anymore, maybe just do `TokenAddress::new(contract_address, token_id).to_string()` everywhere instead